pkg/es/mock_transport.go

```diff
@@ -0,0 +1,11 @@
+package es
```

Comment:
If mockTransport will be used only in the es package, we can also append `_test` to the file, so that the code can be accessed **only** in tests.

Reply:
I don't think this should be leaked outside this package. will add the suffix